alacritty_terminal/src/term/color.rs

```diff
@@ -11,82 +10,56 @@ use alacritty_config_derive::SerdeReplace;
 
 use crate::ansi::NamedColor;
 
-/// Number of terminal colors.
-pub const COUNT: usize = 269;
-
+// Shim around [`vte::ansi::Rgb`] for custom deserialization and
+// serde replace impl.
 #[derive(SerdeReplace, Debug, Eq, PartialEq, Copy, Clone, Default, Serialize)]
-pub struct Rgb {
-    pub r: u8,
-    pub g: u8,
-    pub b: u8,
-}
+pub struct Rgb(vte::ansi::Rgb);
```

Comment:
VTE's rgb should be imported as `VteRgb` at the top.

Reply:
Fixed in https://github.com/alacritty/alacritty/pull/6896/commits/d067434b45ddb26b5dd181a98d6fc022962a9a51.